backend/src/data/repository/user.ts

```diff
@@ -33,7 +21,15 @@ export class UserRepository {
   // Friends
 
   async getFriendsById(userId: DTO.ID): Promise<DTO.IUserDoc | null> {
-    const data = await User.findById(userId).populate('friends', ['firstName', 'lastName', 'photo', 'birthday', 'country', 'speak', 'learn']);
+    const data = await User.findById(userId).populate('friends', [
+      'firstName',
+      'lastName',
+      'photo',
+      'birthday',
+      'country',
+      'speak',
+      'learn'
```

Comment:
Another comment here - at line 46 you already have object with all these props. 
Move to the const kinda UserPattern or something and use here Object.keys(UserPattern), so it is dynamic